tests/Workflow.py

```diff
@@ -81,3 +81,39 @@ def testGetRunsWithStrings(self):
             lambda r: r.id,
             [109950033, 108817672, 108794468, 107927403, 105213061, 105212023],
         )
+
+    def testCreateDispatchWithBranch(self):
+        dispatch_inputs = {"logLevel": "Warning", "message": "Log Message"}
+        workflow = self.g.get_repo("wrecker/PyGithub").get_workflow(
+            "manual_dispatch.yml"
+        )
+        branch = self.g.get_repo("wrecker/PyGithub").get_branch(
+            "workflow_dispatch_branch"
+        )
+
```

Comment:
Would you mind dropping the vertical white space here and in the other tests you've added?

Reply:
Done.